src/main/java/com/gamegoo/service/member/RiotService.java

```diff
@@ -68,14 +105,16 @@ public void updateMemberRiotInfo(String game_name, String tag, String email) {
                 });
 
         //    (2) Champion id, Member id 엮어서 MemberChampion 테이블에 넣기
-        recentChampionIds
+        top3Champions
                 .forEach(championId -> {
+                    Champion champion = championRepository.findById(Long.valueOf(championId))
+                            .orElseThrow(() -> new MemberHandler(ErrorStatus.CHAMPION_NOT_FOUND));
+
                     MemberChampion memberChampion = MemberChampion.builder()
                             .member(member)
```

Comment:
MemberChampion <-> Member가 서로 양방향 연관관계를 맺고 있어서 builder에 바로 member 엔티티를 넣지 않고 
```suggestion
                    MemberChampion memberChampion = MemberChampion.builder()
                        .champion(champion)
                        .build();
                    memberChampion.setMember(member);
```
이렇게 연관관계 메소드를 직접 호출해주는 게 좋을 것 같습니다 !